sbt-bridge/src/xsbt/ConsoleInterface.scala

```diff
@@ -51,6 +33,6 @@ class ConsoleInterface {
       } ++
```

Comment:
Try matching the signature [here](http://www.scala-sbt.org/1.0.0/api/xsbt/ConsoleInterface.html). That might fix your failing tests.

Reply:
I changed the signature of `run` to match. Still no luck :(